.gitlab-ci.yml

```diff
@@ -54,6 +54,36 @@ tox:
     - !reference [.load_github_key, script]
     - tox -e py
 
+# used by release
+.python_versions:
+  parallel:
+    matrix:
+      - BUILD_PY_VERSION: [python3.6]
+      - RELEASE_PY_VERSION: [python3.6]
```

Comment:
Shouldn't we add more versions? (... 3.8, 3.9+?)
Doesn't it have to match the builder below?

Reply:
If I understand correctly, this build version could be used by python3.6+